src/middlewared/middlewared/etc_files/local/nut/ups_perms.py

```diff
@@ -20,6 +20,9 @@ def ups_config_perms(middleware):
         if ups_config['mode'].lower() != 'master':
             os.remove(file)
 
+    if os.path.exists(UPS_POWERDOWN_FLAG_FILE):
```

Comment:
Why not pathlib.Path(UPS_POWERDOWN_FLAG_FILE).unlink(missing_ok=True)?

Reply:
No reason, this does the job as well :D

Reply:
This is prone to race condition (which will never occur, but still)